app/controllers/api/v1/court_hearings_controller.rb

```diff
@@ -0,0 +1,36 @@
+module Api
+  module V1
+    class CourtHearingsController < ApiController
+      def create
+        court_hearing = CourtHearing.create!(court_hearings_attributes)
+
+        render json: court_hearing, status: :created
+      end
+
+    private
+
+      def court_hearings_attributes
+        court_hearings_params.merge(move: move)
+      end
+
+      def court_hearings_params
+        params.require(:data).require(:attributes).permit(
+          :start_time,
+          :case_start_date,
+          :nomis_case_number,
+          :nomis_case_id,
+          :case_type,
+          :comments,
+        )
+      end
+
+      def move
```

Comment:
Is it possible to add this relationship to the swagger docs?

Reply:
This is fixed I believe, yeah